mkdocs.yml

```diff
@@ -34,13 +34,16 @@ nav:
           - 'Firmware update': variants/dell_optiplex/firmware-update.md
           - 'Recovery': variants/dell_optiplex/recovery.md
       - 'Asus KGPE-D16':
-          - 'Releases': variants/asus_kgpe_d16/releases.md
-          - 'Hardware setup': variants/asus_kgpe_d16/setup.md
-          - 'Building manual': variants/asus_kgpe_d16/building-manual.md
-          - 'Hardware configuration matrix': variants/asus_kgpe_d16/hardware-matrix.md
+          - 'Overview': variants/asus_kgpe_d16/overwiew.md
```

Comment:
The `overview.md` section has been linked incorrectly. Please, improve this.

Reply:
Done